tooling/FST-1004-versioning-plan.md

```diff
@@ -0,0 +1,88 @@
+# F# Tooling RFC FST-1004 - Versioning
+
+We have strong motivation to decouple versioning of the F# compiler and tools from the F# language and core library. The reasoning is as follows:
+
+* The F# major language version number will change more and more slowly over time.  Proposed and approved new features still fit snugly in the minor version number.
+* The F# minor language version is expected to change, but not as often as tooling and SDK versions.
+* Decoupling the FSharp.Compiler.Service version from the language version has proven to be a good thing, as it has allowed the adoption of SemVer and rapid iteration.  A similar model can be applied for the F# compiler and tooling.
+* Coupling the compiler version with the language version is not always done in other languages (e.g., C#), and such decoupling hasn't been a problem.
+
+## Table of versions
+
+| Visual Studio version | 15.5 (current) | 15.6 | 15.7 | vNext |
+|------------|----------:|-----:|------:|------:|
+| **F# language version** | 4.1 | 4.1 | 5.4 | 4.x |
```

Comment:
I like the fact of having new major version of F# so soon, but I think you meant `4.5`, right?

Reply:
Yes, mistype. Thanks!